locale/af/LC_MESSAGES/django.po

```diff
@@ -359,9 +363,11 @@ msgstr "Nee dankie"
 msgid "Contribution made, thank you."
 msgstr ""
 
-#: src/olympia/addons/templates/addons/details_box.html src/olympia/legacy_discovery/templates/legacy_discovery/addons/detail.html
-msgid "No restart required"
-msgstr "Herbegin nie nodig nie"
+#: src/olympia/addons/templates/addons/details_box.html src/olympia/addons/templates/addons/impala/details.html src/olympia/addons/templates/addons/impala/listing/items.html
+#: src/olympia/editors/models.py src/olympia/legacy_discovery/templates/legacy_discovery/addons/detail.html
+#, fuzzy
```

Comment:
No - we'd need to generate `.mo` files with the `--use-fuzzy` flag and we aren't currently. Still, it's good to have so that translators have something to start from.